concrete/src/Tree/Node/Exception/MoveException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+namespace Concrete\Core\Tree\Node\Exception;
+
+class MoveException extends \RuntimeException
+{
+    
+}
```

Comment:
No, I just need to run php-cs-fixer :wink:

Done in https://github.com/concrete5/concrete5/pull/4352/commits/3c1ae12003e74b9d44ca6f1eb8f3901efd1913f3